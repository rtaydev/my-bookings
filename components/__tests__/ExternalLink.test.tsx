import React from "react";
import { render } from "@testing-library/react-native";
import { ExternalLink } from "../ExternalLink";

it("renders correctly", () => {
  const tree = render(
    <ExternalLink href="https://example.com">Link</ExternalLink>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it("has correct props", () => {
  const { getByText } = render(
    <ExternalLink href="https://example.com">Link</ExternalLink>,
  );
  const linkElement = getByText("Link");
  expect(linkElement.props.href).toBe("https://example.com");
});
