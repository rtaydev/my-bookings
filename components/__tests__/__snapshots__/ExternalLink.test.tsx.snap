// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  href="https://example.com"
  onPress={[Function]}
  role="link"
>
  Link
</Text>
`;
